docs/requirements.md

```diff
@@ -0,0 +1,146 @@
+## Hydrophonitor CLI
+
+### Requirements
+
+Introduces a command line utility called `hp-cli` which interfaces with a Hydrophonitor, a Raspberry Pi based hydrophone passive acoustic monitoring system.
+
+### Supported commands
+
+#### 1. Import
+
+Introduces the following command:
+
+```sh
+hp-cli import DEVICE --output-path OUTPUT [options]
+```
+
+```
+Required arguments:
+
+DEVICE                      Path to USB mass storage or SD card where data will
+                            be imported from.
+
+--output-path OUTPUT        Path to where the directory for imported data
+                            will be created and data will be imported.
+
+Supported options:
+
+--merge-wavs                Merge all wav files in one deployment into one wav file.
+                            Resulting file will have as its name <timestamp>_audio.wav
+                            where the timestamp is the timestamp of the first wav file.
+
+--no-metadata               Do not prompt the user for submitting metadata about a deployment.
+
+--import-path PATH          Alternative path on the device file system from which data should
+                            be imported. By default, data is imported from /output.
+
+--help                      Displays help for available commands. All other arguments are ignored.
+
+--verbose                   Increase CLI verbosity.
+```
+
+This command imports data from the given device, creating a new directory for each imported deployment at the specified output path. Data import is done with `rsync`, meaning that the user can interrupt the import and resume it by rerunning the import command. This command does not remove files from the device.
+
+At the beginning of the import, the CLI lists the deployments it has detected. For each deployment, the size of the deployment folder is printed. A progress bar shows the progress of the import.
+
+Upon successful import of a single deployment, a metadata file is created. CLI asks user to fill in optional information unless ``--no-metadata`` is supplied. 
+
+The device is attached to the host computer as USB mass storage. It is assumed that the output data is located in `/output` directory at the root of the device filesystem. This directory contains a timestamped directory for each deployment (one boot from startup to shutdown).
+
+```
+/output/
+    <deployment-timestamp>/
+        audio/
+            <timestamp>_audio.wav
+            ...
+        gps/
+            <timestamp>_gps.json
+            ...
+        depth/
+            <timestamp>_depth.csv
```

Comment:
Depth recording should only have one .csv file.

Reply:
I think we should have the option to merge multiple depth csv files in case the service has restarted during the deployment and started writing to a new file.